R/gh.R

```diff
@@ -65,7 +65,7 @@
 #' @param .params Additional list of parameters to append to `...`.
 #'   It is easier to use this than `...` if you have your parameters in
 #'   a list already.
-#'
+#' @param .max_wait Maximum number of minutes to wait if rate limited.
```

Comment:
@gaborcsardi what do you think of this name, units (i.e. minutes not seconds), and default value?

Reply:
I agree that people usually want minutes, but we usually use seconds, so I would use seconds here as well. But it is not a very strong opinion.